src/styles/colors.js

```diff
@@ -35,6 +36,7 @@ export default {
     lightDefaultButton: '#EEEBE7',
     lightDefaultButtonHover: '#E3DFD9',
     lightDefaultButtonPressed: '#D2CCC3',
+    lightRightHandPanelOverlay: '#EBE6DF',
```

Comment:
We shouldn't have a difference between dark and light. 

Also, the overlay color is actually just our `colors.border` at 72%. So we might just want to reference the border color here as well rather than stating a new value. 

cc @Expensify/design